uarch-bench.sh

```diff
@@ -4,41 +4,15 @@
 
 set -e
 
-# pass cpu # and get the state of the turbo mode, 0 is enabled, 1 is disabled
-function get_core_turbo_intel {
-	echo $(sudo rdmsr -p${core} 0x1a0 -f 38:38)
-}
-
-# get the turbo state of all cores as a string
-function get_turbo_state_string {
-	local turbo_state
-	for core in $cores; do
-    	turbo_state+="$(get_core_turbo_intel $core) "
-	done
-	echo $turbo_state
-}
-
-
-function is_even {
-  if [ $(( $1 % 2)) -eq 0 ]; then echo "O";
-  else echo "X";
-  fi
-}
-
 export VENDOR_ID=$(lscpu | grep 'Vendor ID' | egrep -o '[^ ]*$')
 export MODEL_NAME=$(lscpu | grep 'Model name' | sed -n 's/Model name:\s*\(.*\)$/\1/p')
 export SCALING_DRIVER=$(cat /sys/devices/system/cpu/cpu0/cpufreq/scaling_driver)
 export SCALING_GOVERNOR=$(cat /sys/devices/system/cpu/cpu0/cpufreq/scaling_governor)
+export NO_TURBO=$(cat /sys/devices/system/cpu/intel_pstate/no_turbo)
```

Comment:
See the review overall comments, but we should handle the case where this path doesn't exist.